### PR TITLE
Add versionless value for admin api and terraform

### PR DESCRIPTION
**Pull request by mirnawong1**
This pr adds clarification on which value to input for dbt_version for the admin api and terraform. Also raised in community slack. More context in [internal slack thread](https://dbt-labs.slack.com/archives/C05FWBP9X1U/p1720107316597949?thread_ts=1715782793.330549&channel=C05FWBP9X1U&message_ts=1720107316.597949)

### CODE REVIEW

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ dbt Cloud is going "versionless." This means you'll automatically get early acce
 
 Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
 
+For the dbt Cloud Admin API and Terraform, use the value `versionless` to configure your environment.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]: not sure if I should specify that users can input the value in the dbt_version key ?

Comment (jtcohen6) [reply website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]: Yes, I think that's a good clarification - "To upgrade an environment via Admin API or Terraform, set `dbt_version` to the string `"versionless"`

Comment (mirnawong1) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
To upgrade an environment in the dbt Cloud Admin API or Terraform, set `dbt_version` to the string `versionless`.
```

File: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ By choosing to **Keep on latest version**, you opt for a versionless experience
 
 You can upgrade to **Keep on latest version** and the versionless experience no matter which version of dbt you currently have selected. As a best practice, dbt Labs recommends that you test the upgrade in development first; use the [Override dbt version](#override-dbt-version) setting to test _your_ project on the latest dbt version before upgrading your deployment environments and the default development environment for all your colleagues.
 
+For the dbt Cloud Admin API and Terraform, use the value `versionless` to configure your environment.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md]:
```suggestion
To upgrade an environment in the dbt Cloud Admin API or Terraform, set `dbt_version` to the string `versionless`.
```

**Comment by mirnawong1** [comment]
Thank you @jtcohen6, I've folded in your feedback

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

This is fab! It's very clear and concise. I've approved this for you. Please let me know if there's anything else I can do to help out with this.

Kind Regards

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 3]
```diff
@@ -21,6 +21,8 @@ dbt Cloud is going "versionless." This means you'll automatically get early acce
 
 Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
 
+To upgrade an environment in the dbt Cloud Admin API or Terraform, set `dbt_version` to the string `versionless`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
To upgrade an environment in the [dbt Cloud Admin API](/docs/dbt-cloud-apis/admin-cloud-api) or [Terraform](https://registry.terraform.io/providers/dbt-labs/dbtcloud/latest), set `dbt_version` to the string `versionless`.
```

File: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ By choosing to **Keep on latest version**, you opt for a versionless experience
 
 You can upgrade to **Keep on latest version** and the versionless experience no matter which version of dbt you currently have selected. As a best practice, dbt Labs recommends that you test the upgrade in development first; use the [Override dbt version](#override-dbt-version) setting to test _your_ project on the latest dbt version before upgrading your deployment environments and the default development environment for all your colleagues.
 
+To upgrade an environment in the dbt Cloud Admin API or Terraform, set `dbt_version` to the string `versionless`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md]:
```suggestion
To upgrade an environment in the [dbt Cloud Admin API](/docs/dbt-cloud-apis/admin-cloud-api) or [Terraform](https://registry.terraform.io/providers/dbt-labs/dbtcloud/latest), set `dbt_version` to the string `versionless`.
```

**Review by jtcohen6 — APPROVED** [review]
thanks @mirnawong1!

**Comment by mirnawong1** [comment]
great stuff! thanks for reviewing @jtcohen6 and @nataliefiann - merging this now!

**Comment by mirnawong1** [comment]
preview looks fine: https://docs-getdbt-8opkei94f-dbt-labs.vercel.app/docs/dbt-versions/core-upgrade/upgrading-to-v1.8

**Pull request merged**